### PR TITLE
[Barefoot] Enable 'TELEMETRY_WRITABLE' for telemetry container

### PR DESCRIPTION
**Pull request by YuliiaBashko**
#### Why I did it
For testing `gNMI/gNOI` we need to enable r/w mode for telemetry container

#### How I did it
Added `TELEMETRY_WRITABLE=y` option to `make` command
#### How to verify it
Build SONiC with command `make TELEMETRY_WRITABLE=y target/sonic-aboot-barefoot.swi`
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

**Comment by YuliiaBashko** [comment]
@xumia Could you please take a look?

File: .azure-pipelines/azure-pipelines-build.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ jobs:
             docker_syncd_rpc_image: yes
             platform_rpc: bfn
             swi_image: yes
+            BUILD_OPTIONS: $(BUILD_OPTIONS) TELEMETRY_WRITABLE=y
```
Comment (xumia) [inline .azure-pipelines/azure-pipelines-build.yml:51]: Do you mean as below?
BUILD_OPTIONS: ${{ parameters.buildOptions }} TELEMETRY_WRITABLE=y

Comment (YuliiaBashko) [reply .azure-pipelines/azure-pipelines-build.yml:51]: I see, thanks, I\`ll fix this. So it means that my change wouldn\`t  work at all?

**Review by akokhan — APPROVED** [review]

**Comment by YuliiaBashko** [comment]
@xumia Could you please check my new change?

**Comment by lguohan** [comment]
telemtry_writable should be consistent across different platform. we cannot enable write only on a particular platform.

**Comment by lguohan** [comment]
that is not the current plan. the viable solution is to make the telemetry writable as a runtime option, then user can enable it on the fly and no longer a build option.

**Pull request closed without merging**